src/sentry/api/issue_search.py

```diff
@@ -38,6 +41,8 @@
 for status_key, status_value in STATUS_QUERY_CHOICES.items():
     is_filter_translation[status_key] = ("status", status_value)
 
+for substatus_key, substatus_value in SUBSTATUS_UPDATE_CHOICES.items():
```

Comment:
I'm not familiar enough with this stuff to know: is there a reason one of these is query choices and one is update choices? Do they actually serve different purposes?

Reply:
I think I just ended up naming `SUBSTATUS_UPDATE_CHOICES` based on what I was using it for at the time. They should serve the same purpose.